### PR TITLE
Have Github Actions also run tests

### PR DESCRIPTION
**Pull request by jonesmz**

### CODE REVIEW

File: .github/workflows/release-ninja-binaries.yml [hunk 1]
```diff
@@ -40,6 +40,7 @@ jobs:
         mkdir build && cd build
         cmake -DCMAKE_BUILD_TYPE=Release ..
         cmake --build . --parallel --config Release --target ninja
+        ctest -vv
```
Comment (jhasse) [inline .github/workflows/release-ninja-binaries.yml]: Will this print the output if a test fails?

File: .github/workflows/release-ninja-binaries.yml [hunk 2]
```diff
@@ -40,6 +40,7 @@ jobs:
         mkdir build && cd build
         cmake -DCMAKE_BUILD_TYPE=Release ..
         cmake --build . --parallel --config Release --target ninja
+        cmake --build . --parallel --config Release --target ninja_test
```
Comment (jhasse) [inline .github/workflows/release-ninja-binaries.yml]: Why not remove `--target ninja` from the previous command? Should also be faster.

File: .github/workflows/release-ninja-binaries.yml (hunk 1 above)
Comment (jonesmz) [reply .github/workflows/release-ninja-binaries.yml]: It should, yes.

File: .github/workflows/release-ninja-binaries.yml (hunk 2 above)
Comment (jonesmz) [reply .github/workflows/release-ninja-binaries.yml]: Alright.

**Pull request merged**